microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -432,7 +432,13 @@ To control Image Enhancement, go to `edge://settings/privacy` and turn on or off
 
 Microsoft Edge offers an interactive and seamless experience when you launch the browser for the first time. You can import your browser data to Microsoft Edge from another browser. The data includes your favorites, browsing history, cookies, autofill data, extensions, settings, and other browsing data.
 
-With your confirmation, Microsoft Edge imports browser data from other browsers such as Google Chrome, Mozilla Firefox, or Internet Explorer. Microsoft Edge imports data from your most used browser as defined by your operating system.  If you choose to regularly import your browsing data, browsing data will be imported each time Microsoft Edge is launched.  Importing your data is completed locally on your device, and is stored locally, and is not sent to Microsoft unless you sign in and sync your browsing data.
+With your confirmation, Microsoft Edge imports browser data from other browsers such as Google Chrome, Mozilla Firefox, or Internet Explorer. Microsoft Edge imports data from your most used browser as defined by your operating system.  If you choose to regularly import your browsing data, browsing data will be imported each time Microsoft Edge is launched.
+
+Importing your data is completed locally on your device and is stored locally.  The imported data is sent to Microsoft only after you do at least one of the following actions: 
+
+* Sign into Microsoft Edge with a Microsoft account, and turn on sync'ing of your browsing data across all signed-in devices.
+
+* Allow personalization of experiences and advertising on [Bing](https://www.bing.com), Microsoft News, and other Microsoft services.  To view or delete the imported browsing history that's available for personalization, along with Microsoft Edge browsing history, go to the [Microsoft privacy dashboard](https://account.microsoft.com/privacy/).
```

Comment:
1) Please add directions for how to opt-out of personalization on Bing.
2) For clarity, I suggest starting a new paragraph for how to view/delete imported data.
3) This link "[Microsoft Privacy Dashboard]" takes me to myaccount.microsoft.com and makes me resign into my account. Once i land on that page, it still is not obvious how to view/delete the imported data. I suggest using a more direct link to this data. I also suggest including step by step instructions to delete the data due to the ambiguity of the link.

```suggestion
* Allow personalization of experiences and advertising on [Bing](https://www.bing.com), Microsoft News, and other Microsoft services.  

To view or delete the imported browsing history that's available for personalization, along with Microsoft Edge browsing history, go to the [Microsoft privacy dashboard](https://account.microsoft.com/privacy/).
```

Reply:
separated paragraphs

Reply:
Dedicated [Comment](https://github.com/MicrosoftDocs/edge-developer/pull/2934/files#r1410053855) for Point 1
Dedicated [Comment](https://github.com/MicrosoftDocs/edge-developer/pull/2934/files#r1410055033) for Point 3